src/ui/flutter_app/lib/common/config.dart

```diff
@@ -100,119 +100,131 @@ class Config {
   static int endgameNMoveRule = 100;
   static bool threefoldRepetitionRule = true;
 
+  // TODO: use jsonSerializable
```

Comment:
Will it cause new APK installers to lose their old configuration?

Reply:
JsonSerializable is just a code generator so the parsing won't need to be implemented by hand.
I'm already working on implementing this now.

It wont cause any problems as I'll specify the JSON Key that the current version has.